jina/clients/python/__init__.py

```diff
@@ -24,7 +26,7 @@
     import argparse
 
 
-class PyClient(GrpcClient):
+class PyClient(AsyncGrpcClient):
```

Comment:
What is the difference between this `client` and `PyClientRuntime` or `PyClientPea`?

Reply:
Is this the `real` client used in regular Grpc calls like in examples when using `input_lines` etc.. and the other is internal and instantiated when we expose a RestGateway?

Reply:
`PyClient` is the actual client that does the `index` etc operations. But every operation now is `async`. To avoid these in the main process, `PyClientRuntime` allows us to run `PyClient` in a separate process.

Reply:
Ok I see, PyClient is kind of the executor of the regular BasePea (in terms of where is initialized and so on)